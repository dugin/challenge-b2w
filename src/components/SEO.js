import React from 'react';
import {Helmet} from "react-helmet";


const SEO = () => {

    return (
        <Helmet>
            <title>B2W-Bit Challenge</title>
            <meta name="description" content="A frontEnd challenge"/>
            <meta name="author" content="Rodrigo Dugin"/>
            <meta property="og:type" content="challenge"/>
        </Helmet>
    )
};
export default SEO;