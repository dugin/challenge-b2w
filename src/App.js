import React, {PureComponent} from 'react';
import Home from "./pages/Home";
import Footer from "./components/Footer";
import SEO from "./components/SEO";


class App extends PureComponent {

    render() {
        return (
            <div>
                <SEO/>
                <Home/>
                <Footer/>
            </div>
        );
    }
}

export default App;
