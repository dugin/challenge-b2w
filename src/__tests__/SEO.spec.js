import React from 'react';
import toJson from 'enzyme-to-json';
import {shallow} from 'enzyme';
import App from "../App";
import SEO from "../components/SEO";

describe('App Component', () => {

    const seo = shallow(<SEO/>);

    it('renders properly', () => {
        expect(toJson(seo)).toMatchSnapshot();
    });
});