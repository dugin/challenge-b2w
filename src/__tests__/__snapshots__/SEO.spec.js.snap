// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component renders properly 1`] = `
<HelmetWrapper
  defer={true}
  encodeSpecialCharacters={true}
>
  <title>
    B2W-Bit Challenge
  </title>
  <meta
    content="A frontEnd challenge"
    name="description"
  />
  <meta
    content="Rodrigo Dugin"
    name="author"
  />
  <meta
    content="challenge"
    property="og:type"
  />
</HelmetWrapper>
`;
